clang/lib/Format/WhitespaceManager.cpp

```diff
@@ -14,9 +14,81 @@
 #include "WhitespaceManager.h"
 #include "llvm/ADT/STLExtras.h"
 
+#include <fstream>
```

Comment:
As far as I see in LLVM code, they don't group includes (and don't make empty lines between groups)

Reply:
feel free to change